contracts/JBDirectory.sol

```diff
@@ -32,6 +36,14 @@ contract JBDirectory is IJBDirectory, JBOperatable {
   */
   mapping(uint256 => mapping(address => IJBTerminal)) private _primaryTerminalOf;
 
+  /**
+    @notice
+    Addresses that can change a project's controllers. Those are controllers
+    that have been vetted and verified by Juicebox owners as well as custom
+    launching project contracts
```

Comment:
Consider: Addresses that can set a project's controller. These addresses/contracts have been vetted and verified by Juicebox owners.